targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/TARGET_DISCO_L496AG/PinNames.h

```diff
@@ -285,8 +285,8 @@ typedef enum {
 #endif
 
     // Generic signals
-    LED1        = PB_13, // Green LD2 on board
-    LED2        = LED1,
+    LED1        = PA_5,  // Green LD3 on board
+    LED2        = PB_13, // Green LD2 on board --> Warning: the state is reverted (1=OFF and 0=ON)
```

Comment:
Not coherent with PeripheralPins.c:
    {PB_13,      UART_3,  STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF7_USART3)}, // Connected to LED1

Reply:
Forgot the comments in this file. I will update it too. Thanks.